app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,20 @@
+class LikesController < ApplicationController
+  # def create
+  #  @post = Post.find(params[:id])
+  # @like = @post.likes.new
+  # @like.post_id = @post.id
+  # @like.author_id = @user.id
+  # @like.save
+  # redirect_to "/users/#{current_user.id}/posts/#{@post.id}"
+  # end
```

Comment:
- [ ] @V0pe kindly remove the commented code, it's not good practice to have commented code in the codebase.

Reply:
@danmainah Great eye. I will adjust that. Thanks